### PR TITLE
Clean up FAQ, add DYP Finance as a partner

### PR DESCRIPTION
**Pull request by best-coder-NA**
Screen Shot 2021-07-15 at 1 05 58 AM

### CODE REVIEW

File: src/routes/faq.svelte [hunk 1]
```diff
@@ -22,225 +22,168 @@
...
     <FAQItem>
-      <span slot="question">What is the expert mode? How can I toggle it?</span>
+      <span slot="question">What exchanges is PNG listed on?</span>
       <span slot="answer">
-        Expert mode bypasses confirmation modals and allows high slippage trades. So it's not recommended for
-        unexperienced users. You can toggle the expert mode by clicking on settings at the
-        <a class="hover:no-underline" href="https://app.pangolin.exchange/#/swap">swap page</a>
-        Use at your own risk.
+        PNG is currently listed on
+        <a class="hover:no-underline" href="https://www.gate.io/en/trade/PNG_USDT">Gate.io</a>
```
Comment (dhrubabasu) [inline src/routes/faq.svelte:76]:
```suggestion
        A list of exchanges supporting PNG can be found
        <a class="hover:no-underline" href="https://coinmarketcap.com/currencies/pangolin/markets/">here</a>.
```

File: src/routes/faq.svelte [hunk 2]
```diff
@@ -22,225 +22,168 @@
...
     <FAQItem>
-      <span slot="question">Can I send my Avalanche assets from Metamask to an Exchange or an ETH address?</span>
+      <span slot="question">Who is behind Pangolin Exchange?</span>
       <span slot="answer">
-        No, you cannot send assets across networks from Avalanche to Ethereum or vice versa. You need to use the <a
-          class="hover:no-underline"
-          href="https://aeb.xyz/#/transfer">Avalanche-Ethereum Bridge</a
-        > every time you want to send assets between networks.
+        Pangolin was originally created by AvaLabs. Eventually, the project was transitioned to a community-hired core
+        team.
```
Comment (dhrubabasu) [inline src/routes/faq.svelte:41]:
```suggestion
        Pangolin is a community-driven project. The initial development work was done by Ava Labs. A community-hired core team are now the lead developers.
```

File: src/routes/faq.svelte [hunk 3]
```diff
@@ -22,225 +22,168 @@
...
     <FAQItem>
-      <span slot="question">How can I see the top pairs on Pangolin?</span>
+      <span slot="question">How can I see the pools offered in Pangolin and their APYs?</span>
       <span slot="answer">
-        Go to the analytics page <a class="hover:no-underline" href="https://info.pangolin.exchange/">here</a> to see top
-        pairs on Pangolin.
+        Go to the New PNG page <a class="hover:no-underline" href="https://app.pangolin.exchange/#/png/1">here</a> to see
+        all the pools currently offered in Pangolin along with their APYs. Keep in mind that the APY for a specific pool
+        will go down as users bring in more liquidity. (and vice versa) Also, Pangolin has limited the pools to PNG and AVAX
+        pairs only.
```
Comment (dhrubabasu) [inline src/routes/faq.svelte:111]:
```suggestion
        Go to the New PNG page <a class="hover:no-underline" href="https://app.pangolin.exchange/#/png/1">here</a> to see
        all the PNG-incentivized pools offered in Pangolin along with their current APYs.
```

File: src/routes/faq.svelte [hunk 4]
```diff
@@ -22,225 +22,168 @@
...
     </FAQItem>
 
     <FAQItem>
-      <span slot="question">How can I see the top tokens on Pangolin?</span>
+      <span slot="question">Are there any risks involved in being a liquidity provider on Pangolin?</span>
       <span slot="answer">
-        Go to the analytics page <a class="hover:no-underline" href="https://info.pangolin.exchange/">here</a> to see the
-        top tokens on Pangolin.
+        Yes, IL (impermanent loss) is one of the biggest risks of being a LP on any decentralized exchange. In terms of
+        security, Pangolin is a fork of Uniswap and Sushiswap: It uses the same code for the smart contracts.
```
Comment (dhrubabasu) [inline src/routes/faq.svelte:119]:
```suggestion
		No, Pangolin is a fork of Uniswap V2. You can find all our contracts <a class="hover:no-underline" href="https://github.com/pangolindex/exchange-contracts">here</a>.
		By being a liquidity provider, your overall position may decrease in value due to impermanent loss. Please do your own research before providing liquidity.
```

**Review by dhrubabasu — CHANGES_REQUESTED** [review]
Looks pretty good, made some wording adjustments.

**Comment by best-coder-NA** [comment]
Added your changes @dhrubabasu thanks. Merging

**Pull request merged**